data_validation/combiner.py

```diff
@@ -24,8 +24,10 @@
 import logging
 import ibis
 import ibis.expr.datatypes as dt
+import ibis.expr.types as types
```

Comment:
If this is imported just for type hints then we should do it inside a TYPE_CHECKING condition. Search for TYPE_CHECKING in the repo for an example.